README.adoc

```diff
@@ -12,8 +12,10 @@ the ports not yet supported in the 5.0 release you must use the 4.14 release.
 
 The initial release of OCaml 5.0 only supports the native compiler under ARM64
 and x86-64 architectures under Linux, macOS and the BSDs. On Windows, only the
-MinGW-w64 port is supported. Native code support for Linux on RISC-V and
-s390x/IBM Z will be available in OCaml 5.1, and Linux on Power in OCaml 5.2.
+MinGW-w64 port is supported in OCaml 5.0 and Cygwin is restored in 5.1. On Linux,
```

Comment:
`the Cygwin port`? Otherwise, it sounds to me like something happened to `Cygwin` itself?

Reply:
Fixed and rebased.